packages/tasks/src/hardware.ts

```diff
@@ -108,6 +108,10 @@ export const SKUS = {
 				tflops: 30.6,
 				memory: [12, 10],
 			},
+			"RTX 3070 Ti Laptop": {
+				tflops: 16.60,
```

Comment:
```suggestion
				tflops: 16.6,
```

Reply:
Sorry about that... I should have pulled down the source and followed the flow, including linting - I was lazy and just used the web editor. Thanks for the fix.